<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>781</width>
      <height>581</height>
     </rect>
    </property>
    <widget class="QTableView" name="tableView">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>761</width>
       <height>461</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="back_btn">
     <property name="geometry">
      <rect>
       <x>670</x>
       <y>10</y>
       <width>51</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Back</string>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>761</width>
       <height>71</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="add_btn">
          <property name="styleSheet">
           <string notr="true">font: 20pt &quot;MS Shell Dlg 2&quot;;</string>
          </property>
          <property name="text">
           <string>ADD STUDENT</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="remove_btn">
          <property name="styleSheet">
           <string notr="true">font: 20pt &quot;MS Shell Dlg 2&quot;;</string>
          </property>
          <property name="text">
           <string>Remove STUDENT</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="import_btn">
          <property name="styleSheet">
           <string notr="true">font: 20pt &quot;MS Shell Dlg 2&quot;;</string>
          </property>
          <property name="text">
           <string>Import STUDENT</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="export_btn">
          <property name="styleSheet">
           <string notr="true">font: 20pt &quot;MS Shell Dlg 2&quot;;</string>
          </property>
          <property name="text">
           <string>Export STUDENT</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="save_btn">
        <property name="text">
         <string>Save Data</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
